routes.go

```diff
@@ -20,79 +20,82 @@ func setupRoutes(e *echo.Echo) {
-	v3.POST("/applications/:id/pause", ApplicationPause, middleware.Tenancy)
-	v3.POST("/applications/:id/unpause", ApplicationUnpause, middleware.Tenancy)
-
-	// Authentications
-	v3.GET("/authentications", AuthenticationList, tenancyWithListMiddleware...)
-	v3.GET("/authentications/:uid", AuthenticationGet, middleware.Tenancy)
-	v3.POST("/authentications", AuthenticationCreate, permissionMiddleware...)
-	v3.PATCH("/authentications/:uid", AuthenticationUpdate, permissionMiddleware...)
-	v3.DELETE("/authentications/:uid", AuthenticationDelete, permissionMiddleware...)
-
-	// ApplicationTypes
-	v3.GET("/application_types", ApplicationTypeList, listMiddleware...)
-	v3.GET("/application_types/:id", ApplicationTypeGet)
-	v3.GET("/application_types/:application_type_id/sources", ApplicationTypeListSource, tenancyWithListMiddleware...)
-
-	// Endpoints
-	v3.GET("/endpoints", EndpointList, tenancyWithListMiddleware...)
-	v3.GET("/endpoints/:id", EndpointGet, middleware.Tenancy)
-	v3.POST("/endpoints", EndpointCreate, permissionMiddleware...)
-	v3.DELETE("/endpoints/:id", EndpointDelete, permissionMiddleware...)
-	v3.GET("/endpoints/:endpoint_id/authentications", EndpointListAuthentications, tenancyWithListMiddleware...)
-
-	// ApplicationAuthentications
-	v3.GET("/application_authentications", ApplicationAuthenticationList, tenancyWithListMiddleware...)
-	v3.GET("/application_authentications/:id", ApplicationAuthenticationGet, middleware.Tenancy)
-	v3.GET("/application_authentications/:application_authentication_id/authentications", ApplicationAuthenticationListAuthentications, tenancyWithListMiddleware...)
-	v3.POST("/application_authentications", ApplicationAuthenticationCreate, permissionMiddleware...)
-	v3.DELETE("/application_authentications/:id", ApplicationAuthenticationDelete, permissionMiddleware...)
-
-	// AppMetaData
-	v3.GET("/app_meta_data", MetaDataList, listMiddleware...)
-	v3.GET("/app_meta_data/:id", MetaDataGet)
-	v3.GET("/application_types/:application_type_id/app_meta_data", ApplicationTypeListMetaData, listMiddleware...)
-
-	// SourceTypes
-	v3.GET("/source_types", SourceTypeList, listMiddleware...)
-	v3.GET("/source_types/:id", SourceTypeGet)
-	v3.GET("/source_types/:source_type_id/sources", SourceTypeListSource, tenancyWithListMiddleware...)
-
-	// Red Hat Connector Connections
-	v3.GET("/rhc_connections", RhcConnectionList, tenancyWithListMiddleware...)
-	v3.GET("/rhc_connections/:id", RhcConnectionGetById, permissionMiddleware...)
-	v3.POST("/rhc_connections", RhcConnectionCreate, permissionMiddleware...)
-	v3.PATCH("/rhc_connections/:id", RhcConnectionUpdate, permissionMiddleware...)
-	v3.DELETE("/rhc_connections/:id", RhcConnectionDelete, permissionMiddleware...)
-	v3.GET("/rhc_connections/:id/sources", RhcConnectionSourcesList, permissionWithListMiddleware...)
+	apiVersions := []string{"v1.0", "v2.0", "v3.0", "v3.1"}
+	for i := range apiVersions {
```

Comment:
I would use a `for _, version` here since we don't use the index of the loop for anything.

Reply:
Sure, done.